plugin/restore/mysql/mysql.go

```diff
@@ -336,7 +444,7 @@ func (r *Restore) SyncArchivedBinlogFiles(ctx context.Context, instance *api.Ins
 
 		if localBinlogSize != serverBinlog.Size {
 			// exist on local and file size not match, delete and then download it
-			if err := os.Remove(filepath.Join(binlogDir, serverBinlog.Name)); err != nil {
+			if err := os.Remove(filepath.Join(r.binlogDir, serverBinlog.Name)); err != nil {
```

Comment:
Would this be an error or at least we emit a warning log?

Reply:
I prefer to emit a warning log. It's not fatal, we can make it right by redownloading.